clientjs/rxhtml.js

```diff
@@ -3064,6 +3064,41 @@ var RxHTML = (function () {
     }
   };
 
+  transforms['dateFmt'] = function(format, d) {
+    if (typeof (d) == "string") {
+      // only transform strings
+      if (d == "1-01-01") return ""
```

Comment:
magic #? or this is the default date? wonder why it isn't 1-01-001

Reply:
Yep, that's the default date. It's kind of an odd value 🤷‍♂️